docs/concepts/README.md

```diff
@@ -1,4 +1,139 @@
 # Concepts
 
-This section of the documentation helps you learn about the JobSet API and its semantics.
+A JobSet creates one or more Jobs. It allows you to create sets of jobs of different or identical templates, and control their lifecycle together. 
+
+## Running an Example JobSet
+
+Here is an example JobSet. It runs a distributed PyTorch training workload.
+
+```yaml
+apiVersion: jobset.x-k8s.io/v1alpha1
+kind: JobSet
+metadata:
+  name: pytorch
+spec:
+  replicatedJobs:
+    - name: workers
+      template:
+        spec:
+          parallelism: 4
+          completions: 4
+          backoffLimit: 0
+          template:
+           spec:
+            containers:
+            restartPolicy: Never
```

Comment:
```suggestion
            restartPolicy: Never
            containers:
```

Reply:
removed it completely since we now default this